product_tier_validation/models/product_template.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ProductTemplate(models.Model):
+    _name = "product.template"
+    _inherit = ["product.template", "tier.validation"]
+    _tier_validation_manual_config = False
```

Comment:
```suggestion
    _tier_validation_manual_config = False
    _state_to = ["sellable"]
```
Useful for changing draft status to salable status. What do you thing?

Reply:
@CILC98, Thanks for the suggestion! I applied the change